core/src/main/java/io/fabric8/maven/core/handler/ProbeHandler.java

```diff
@@ -46,6 +47,9 @@ public Probe getProbe(ProbeConfig probeConfig)  {
             probe.setTimeoutSeconds(timeoutSeconds);
         }
         HTTPGetAction getAction = getHTTPGetAction(probeConfig.getGetUrl());
+        if(StringUtils.isEmpty(getAction.getHost())) {
```

Comment:
It was a bit of a hack to allow an empty HTTP host to be set in the liveness/readiness checks. Without this minor change an empty host value would result in a literal empty host value being added to the output YAML. For now I've just removed this, as it should have been a part of issue #622.